modules/openapi-generator/src/main/resources/kotlin-client/README.mustache

```diff
@@ -3,11 +3,11 @@
 ## Requires
 
 {{#jvm}}
-* Kotlin 1.4.30
+* Kotlin 1.5.10
```

Comment:
I think the version for the `jvm` variant should remain unchanged. Otherwise it results in changing this version in READMEs for `kotlin-gson`, `kotlin-jvm-okhttp4-coroutines`, `kotlin-moshi-codegen` and other, and in this PR we only deal with the multiplatform variant.